.vib/vib-cypress.json

```diff
@@ -0,0 +1,51 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/chart"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "runtime_parameters": "{CARTO_RUNTIME_PARAMETERS}",
+        "target_platform": {
+          "target_platform_id": "{VIB_ENV_TARGET_PLATFORM}",
+          "size": {
```

Comment:
Can we try with the default instance type? I think would not need this big machine for the target platform.